station/signals/handlers.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from guardian.shortcuts import assign_perm
+
+from station.models import (
+    Basin,
+    Country,
+    Ecosystem,
+    Institution,
+    Place,
+    Region,
+    Station,
+    StationType,
+)
+
+User = get_user_model()
+
+
+@receiver(post_save, sender=Country)
+@receiver(post_save, sender=Region)
+@receiver(post_save, sender=Ecosystem)
+@receiver(post_save, sender=Institution)
+@receiver(post_save, sender=StationType)
+@receiver(post_save, sender=Place)
+@receiver(post_save, sender=Basin)
+@receiver(post_save, sender=Station)
+def set_permissions(sender, instance, **kwargs):
+    """Set object-level permissions"."""
+    # Get permissions for model
+    delete, change, view = _get_perm_codenames(sender)
+
+    # Assign view permissions for all users
+    assign_perm(view, User.objects.all(), instance)
+
+    # Assign change and delete permissions for owner
+    if instance.owner:
+        for perm in [change, delete]:
+            assign_perm(perm, instance.owner, instance)
```

Comment:
I mention this here, but it applies to all other handlers. 

If a permission is not assigned explicitly, does the object have it? What a re the default permissions of an object> In other words, if we create a `Basin`, for example, and not permissions are granted to the owner, what permissions they have? And everyone else? Should those permissions be removed, in addition to grant new ones to the owner?